.github/workflows/uvloop_test.yml

```diff
@@ -38,8 +38,12 @@ jobs:
 
       - name: Test uvloop Extra Install
         run: |
-          python -m pip install -e ".[uvloop]"
+          python -m pip install -e ".[uvloop,d]"
 
-      - name: Primer uvloop run
+      - name: Install test requirements
         run: |
-          black-primer
+          python -m pip install -r test_requirements.txt
+
+      - name: Run unit tests
+        run: |
+          pytest tests
```

Comment:
Will these even hit patched_main() and use uvloop ... I don't think so.

Reply:
Oh, good point. I guess uvloop only kicks in if we use parallelism, so even if I made the test suite use uvloop it wouldn't help much.

Reply:
I'll add a basic test that just runs black on src/, to validate that uvloop mostly works.